docs/topics/strings.md

```diff
@@ -158,27 +158,37 @@ To format a string to your specific requirements, use the [`String.format()`](ht
 function. 
 
 The `String.format()` function accepts a format string and one or more arguments. The format string contains one placeholder 
-(`%`) for each remaining argument, followed by [format specifiers](https://docs.oracle.com/javase/8/docs/api/java/util/Formatter.html#summary). 
-Format specifiers are formatting instructions for the respective argument. In the output, each argument fills its 
-corresponding placeholder in the defined format:
+(`%`) for each remaining argument, followed by format specifiers. 
+Format specifiers are formatting instructions for the respective argument, consisting of flags, width, precision, and 
+conversion type, which collectively shape the output's formatting. Common format specifiers include 
```

Comment:
We can make it more readable by avoiding too many subordinate clauses together. For example, instead of "which collectively shape the output's formatting", have a new sentence "Collectively, format specifiers shape the outputs format".